python/lookout/__init__.py

```diff
@@ -2,4 +2,3 @@
 # DO NOT CHANGE OR ADD ANYTHING HERE
```

Comment:
I think the changes need to be done in https://github.com/src-d/lookout-sdk/blob/master/_tools/protogen_python.sh#L23

Reply:
ohhhh. thanks!
I don't think we should change anything besides generated code from proto when we run protogen.

Reply:
removed here: https://github.com/src-d/lookout-sdk/pull/55/commits/7ba612407f20c69611b59bc3a1830a1339317630